Application/MocapExamples/Plug-in-gait_Simple/Setup/TrialSpecificData_GRFPrediction.any

```diff
@@ -78,33 +78,33 @@ Main.ModelSetup.TrialSpecificData =
 
       HipFlexion = 20;
       HipAbduction = 4;
-      HipExternalRotation = -15;
+      HipExternalRotation = 10;
       KneeFlexion = 8;
       AnklePlantarFlexion = -10;
       SubTalarEversion = 0;
 
-      GlenohumeralFlexion = 0;
-      GlenohumeralAbduction = 1;
-      GlenohumeralExternalRotation = 1;
-      ElbowFlexion = 0;
+      GlenohumeralFlexion = -18;
+      GlenohumeralAbduction = 0;
+      GlenohumeralExternalRotation = 0;
+      ElbowFlexion = 20;

```

Comment:
Is it on purpose that you modified the load-time posture of the models. I think it is fine to improve them, just need to know that it is on purpose :)